src/lnd-actions/send-payment.js

```diff
@@ -44,14 +35,26 @@ const SEND_PAYMENT_DEADLINE = 30
  * @param {SendPaymentRequest} paymentOptions
  * @param {object} opts
  * @param {LndClient} opts.client
+ * @param {Logger} opts.logger
  * @returns {Promise<object>} Resolves with the response from LND
  */
-function sendPayment (paymentOptions, { client }) {
+function sendPayment (paymentOptions, { client, logger }) {
   return new Promise((resolve, reject) => {
-    client.sendPaymentSync(paymentOptions, { deadline: deadline(SEND_PAYMENT_DEADLINE) }, (err, res) => {
-      if (err) return reject(err)
-      return resolve(res)
-    })
+    try {
+      const call = client.router.sendPayment(paymentOptions)
```

Comment:
do we also need to handle `call.on('error', ...)` and `call.on('cancel', ...)` ?